_articles/manage-repos-pop.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: article
+title: Manage Repositories in Pop!_OS
+description: >
+  Add, remove and edit repositories in Pop!_OS
+keywords:
+  - Pop!_OS 18.04
+  - Pop!_OS 19.10
+  - repoman
+  - System76
+  - apt
+image: http://support.system76.com/images/pop-icon.png
+hidden: false
+section: pop-ubuntu
```

Comment:
pop not pop-ubuntu

Reply:
Fixed here : https://github.com/system76/docs/pull/334